btk/metrics.py

```diff
@@ -1,484 +1,110 @@
-    """Initialize column entries of true objects and detection catalog to their
-    default values.
-
-    This is necessary since, if either true objects or detection table is
-    empty, then there is no match in get_detection_match and those column
-    entries would not otherwise be present in the tables.
-
-    Function does not return anything, only the astropy tables are updates.
-    """
-    # initialize true objects table columns
-    num_true = len(true_table)
-    true_table["true_id"] = range(num_true)  # id in blend [0-num_true]
-    # index of blend in test size [0 - len(blend_summary)]
-    true_table["blend_index"] = np.ones(num_true, dtype=int) * (
-        batch_index * batch_size + blend_index
-    )
-    # index of batch in test size [0 - batch_size]
-    true_table["batch_index"] = np.ones(num_true, dtype=int) * batch_index
-    # number of times object was detected [0 - num_det]
-    true_table["num_detections1"] = np.zeros(num_true, dtype=int)
-    true_table["num_detections2"] = np.zeros(num_true, dtype=int)
-    # detection id of closest detection with 2 algorithms [0 - num_det]
-    true_table["closest_det_id1"] = np.ones(num_true, dtype=int) * -1.0
-    true_table["closest_det_id2"] = np.ones(num_true, dtype=int) * -1.0
-    # difference in centroids, i band magnitude and redshift between an object
-    # and its match with algorithm 2
-    true_table["dm_match"] = np.zeros(num_true, dtype=int)
-    true_table["dz_match"] = np.zeros(num_true, dtype=int)
-    true_table["ddist_match"] = np.zeros(num_true, dtype=int)
-    true_table["dnorm_dist_match"] = np.zeros(num_true, dtype=int)
-    # find distance to nearest neighbor. If isolated then set to np.inf
-    true_table["min_dist"] = np.ones(num_true) * np.inf
-    get_closest_neighbor_distance(true_table)
-    # initialize detected objects table columns
-    num_det = len(detected_table)
-    detected_table["detection_id"] = range(num_det)  # id in blend [0-num_det]
-    # index of blend in test size [0 - len(blend_summary)]
-    detected_table["blend_index"] = np.ones(num_det, dtype=int) * (
-        batch_index * batch_size + blend_index
-    )
-    # index of batch in test size [0 - batch_size]
-    detected_table["batch_index"] = np.ones(num_det, dtype=int) * batch_index
-    # id of closest true object; [0 - num_true] if detected, else -1
-    detected_table["match_true_id1"] = np.ones(num_det, dtype=int) * -1
-    detected_table["match_true_id2"] = np.ones(num_det, dtype=int) * -1
-    detected_table["match_galtileid1"] = np.ones(num_det, dtype=int) * -1
-    detected_table["match_galtileid2"] = np.ones(num_det, dtype=int) * -1
+import skimage.metrics
 
 
 def get_detection_match(true_table, detected_table):
     """Match detections to true objects and update values in the input
     blend catalog and detection catalog.
-
     Function does not return anything, only the astropy tables are updated.
```

Comment:
Is this a deprecated comment? Because it seems you are returning `match_table` and then using later in the code.

Reply:
Absolutely, I actually reused this function from the old metrics file and did not change the docstring yet.